test/jdk/javax/swing/JComboBox/DisabledComboBoxFontTestAuto.java

```diff
@@ -113,30 +114,45 @@ private static void testMethod() throws IOException {
                 eColor2 = new Color(enabledImage2.getRGB(x, y));
                 dColor1 = new Color(disabledImage.getRGB(x + 1, y));
                 dColor2 = new Color(disabledImage2.getRGB(x, y));
+            } else if (lafName.equals("Windows")) {
+                // In Windows LAF, the ComboBox sizes are different.
+                // Combo size is 116 x 22 where as Combo2 size is 115 X 20 and
+                // that results in pixel offset of 1px for width and 1px for height.
+                eColor1 = new Color(enabledImage.getRGB(x, y));
+                eColor2 = new Color(enabledImage2.getRGB(x + 1, y - 1));
+                dColor1 = new Color(disabledImage.getRGB(x, y));
+                dColor2 = new Color(disabledImage2.getRGB(x + 1, y - 1));
+            } else if (lafName.equals("GTK+")) {
+                // In GTK LAF, the ComboBox sizes are different.
+                // Combo size is 159 x 30 where as Combo2 size is 179 X 34 and
+                // that results in pixel offset of 10px for width and 2px for height.
+                eColor1 = new Color(enabledImage.getRGB(x, y));
+                eColor2 = new Color(enabledImage2.getRGB(x + 10, y + 2));
+                dColor1 = new Color(disabledImage.getRGB(x, y));
+                dColor2 = new Color(disabledImage2.getRGB(x + 10, y + 2));
```

Comment:
I noticed with these components, sometimes the size varies slightly between different desktop scaling values or resolutions. You have exact combobox pixel sizes in the comments, but does this stay the same when you set your device to 200% or other values?

Maybe best to leave out the exact combo size from the comments regardless.

Reply:
> I noticed with these components, sometimes the size varies slightly between different desktop scaling values or resolutions. You have exact combobox pixel sizes in the comments, but does this stay the same when you set your device to 200% or other values?
Maybe best to leave out the exact combo size from the comments regardless.

Checked with UIScale of 2 and the images are of same size. But your point seems valid that may vary from between different machine for different resolution or scale.
Removed the exact size from the comments.